### ISSUE TITLE
Bot not working on Announcement Channels.

### ISSUE THREAD

**Issue by NxKarim** [atomheartother/A-I-kyan#94]
The bot was suscribed to an "Announcement" channel and was working fine till 2 days ago. But now it doesn't send the tweets or replies to commands if the channel is an Announcement channel.

**Comment by atomheartother** [comment]
Yes, sorry, this is a known regression, QTweet was coded before channels had these subtypes available, and she was never meant to be able to post in them. Recently I coded a few functions to check for channel types and I haven't debugged for announcement channels yet, so they will not work temporarily.

**Comment by atomheartother** [comment]
Attempted fix on the https://github.com/atomheartother/QTweet/tree/feat/newsChannel branch
